MapboxMobileEvents/Categories/NSUserDefaults+MMEConfiguration.m

```diff
@@ -373,6 +373,13 @@ - (BOOL)mme_isCollectionEnabled {
         }
     }
 
+    // Currently storage between App/Extension (AppGroup) is not shared meaning some values such as
+    // privacy review consent are not shared between both. Default collection to OFF until shared support
+    // is implemented
+    if (NSBundle.mme_isExtension) {
```

Comment:
@alfwatt , 
After potential concerns about privacy setting sharing between App/Extension, opted to disable by defaults telemetry gathering in extensions until we can further reach clarity on what should be shared, as well as responsibilities to the developer to enable sharing.